README.md

```diff
@@ -44,11 +45,11 @@ $ docker-compose -f docker-compose-simple.yaml up
 ```
 Note that in this case you have to remove the containers manually before restarting the network.
 
-Starting the network starts an orderer (```orderer```), a peer (```peer```), a chaincode container (```chaincode```), a couchdb container (```couchdb```), and a client container (```cli```).
+Starting the network starts an orderer (```orderer```), a peer (```peer```), a chaincode container (```chaincode```), a couchdb container (```couchdb```), and a command-line-interface container (```cli```).
 
 *  ```chaincode``` is the container compiling and running the chaincode for the peer. For that, a ```build.gradle``` is required in the ```chaincode``` java-project.
-* ```couchdb``` this is used internally to store the ledger state. You probably do not have to worry about this.
-* ```cli``` provides fabric binaries that can be utilized to do all sorts of stuff to the network manually (e.g. query chaincode, add channels, etc.)
+* ```couchdb``` is used internally to store the ledger state. You probably do not have to worry about this.
+* ```cli``` provides fabric binaries that can be utilized to do all sorts of stuff to the network manually (e.g. query chaincode, add channels, register/enroll users etc.)
```

Comment:
You cannot register/enroll users in the dev-network, since it does not run any CA at all.

Reply:
absolutely correct

Reply:
removed in new commit